config/locales/views/criteria/en.yml

```diff
@@ -3,7 +3,7 @@ en:
     new:
       button: "New Criterion"
     delete:
-      checkbox: "Delete Level"
+      checkbox: "Delete Level "
```

Comment:
Remove this whitespace (don't try to fix UI spacing issues by putting spaces in strings, change the CSS if you need to).